packages/connect-form/package.json

```diff
@@ -53,9 +53,10 @@
     "react-dom": "*"
   },
   "dependencies": {
+    "@leafygreen-ui/tabs": "^5.1.5",
```

Comment:
I'm not sure but I think the idea is that we should re-export this in compass-components and depend on this package. /cc @Anemy am I getting this right?

Reply:
Yup that's the idea. It'll help us maintain a single dependency version on leafygreen and make styling tweaks a bit easier if needed. Looks like we already have these tabs exported there: https://github.com/mongodb-js/compass/blob/main/packages/compass-components/src/index.ts#L34

Reply:
Thanks both!